pkgs/by-name/go/gotemplate/package.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGo123Module rec {
   pname = "gotemplate";
-  version = "3.9.2";
+  version = "3.10.1";
 
   src = fetchFromGitHub {
     owner = "coveooss";
     repo = pname;
```

Comment:
Thanks for the review @GaetanLepage.

I fail to see how that may be of any consequence, but I'll apply your suggested change, rebase and force-push (then I'll ask you to kindly review again).

Please don't take this the wrong way, but I'd like to suggest that in the future you provide at least some bit of motivation when asking people to modify their contributions or, in general, put in extra work.

Reply:
You are definitely right. I don't often take the time to see if the author is experienced or not.
In this case, we tend to discourage the use of `pname` in `src` because if someone would rename the package it would break the source fetching.
It is not a big deal but it is still good to do.